imports/ui/views/users/volunteerExperienceList/VolunteerExperienceList.jsx

```diff
@@ -0,0 +1,75 @@
+import React, {Component} from 'react'
+import VolunteerExperienceContent from '/imports/ui/views/users/volunteerExperienceList/VolunteerExperienceContent'
+
+class VolunteerExperienceList extends Component {
+
+  render() {
+    let VolunteerExperienceList = testExperienceList.map((item) => {
+      return <VolunteerExperienceContent key={item.userId} volunteerExperience={item}/>
+
+    })
+
+    return (
+        <div style={styles.listContainer}>
+          <div style={styles.title}>Volunteer Experience</div>
+          <div style={styles.listContainer}>
+            {VolunteerExperienceList}
+          </div>
+        </div>
+    )
+  }
+}
+
+const testExperienceList = [
+  //FIXME: how will we be storing dates?
```

Comment:
The db will just store them as a js date (`new Date()`)

Once they are on the front-end, we can definitely use `moment` or whatever to manipulate them more easily.